hazelcast/src/main/java/com/hazelcast/quorum/impl/QuorumServiceImpl.java

```diff
@@ -106,25 +107,41 @@ public void addQuorumListener(String name, QuorumListener listener) {
         eventService.registerLocalListener(SERVICE_NAME, name, listener);
     }
 
+    /**
+     * Ensures that the quorum for the given operation is present. This requires that there is a quorum configured under the
+     * name which is returned by the operation service.
+     * This in turn requires that the operation is named and that the operation service is a {@link QuorumAwareService}.
+     *
+     * @param op the operation for which the quorum should be present
+     * @throws QuorumException if the operation requires a quorum and the quorum is not present
+     */
     public void ensureQuorumPresent(Operation op) {
         if (inactive) {
             return;
         }
 
-        QuorumImpl quorum = findQuorum(op);
+        final QuorumImpl quorum = findQuorum(op);
```

Comment:
Ah yes. This could not be the place for discussion but working on Marko's code for some time I've come to like them - makes the variables more visible due to syntax highlighting and makes it obvious that the value will not change. I guess I'll keep adding them, maybe just contain it to the new code that I write. I realise that it creates noise when I include this on existing code in a PR.